README.md

```diff
@@ -53,15 +54,16 @@ FMS Model Optimizer is a framework for developing reduced precision neural netwo
     - Nvidia H100 family or higher
     - [llm-compressor](https://github.com/vllm-project/llm-compressor)
 - To enable compute graph plotting function (mostly for troubleshooting purpose):
+    - [matplotlib](https://matplotlib.org/)
     - [graphviz](https://graphviz.org/)
     - [pygraphviz](https://pygraphviz.github.io/)
 
 > [!NOTE]
-> PyTorch version should be < 2.4 if you would like to experiment deployment with external INT8 kernel.
+> PyTorch version should be < 2.5 if you would like to experiment deployment with external INT8 kernel.
```

Comment:
the external kernel is indeed having troubles with PT2.4+ at the moment...

Reply:
ok, rolled back